apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/sql/SqlStore.java

```diff
@@ -0,0 +1,96 @@
+package com.apollographql.android.cache.normalized.sql;
+
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteStatement;
+
+import com.apollographql.android.cache.normalized.CacheStore;
+import com.apollographql.android.cache.normalized.Record;
+
+import javax.annotation.Nullable;
+
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.COLUMN_KEY;
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.COLUMN_RECORD;
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.TABLE_RECORDS;
+
+final class SqlStore extends CacheStore {
```

Comment:
Yeah that's a good point transactions would be a must. Mind making an issue for tracking?   I'm gonna merge this in an hour unless objections. Next piece is going back to the robust parser followed by moving to an external module.